.pre-commit-config.yaml

```
@@ -19,18 +19,24 @@ repos:
       - id: trailing-whitespace
       - id: check-yaml
         exclude: ^recipe/meta.yaml$
+      - id: check-json
+      - id: pretty-format-json
+        args: [--autofix, --no-sort-keys]
+        exclude: ^(menuinst|tests)/_legacy/.*$
       - id: check-merge-conflict
-  - repo: https://github.com/PyCQA/isort
-    rev: 6.0.1
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # Ruff version.
+    rev: v0.11.0
```

```suggestion
    rev: v0.11.2
```

There is a newer version of `ruff`